Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.13-buster as go
```

Comment:
Why are we building on a debian buster based container when we are shipping on an alpine based runtime?
Why not use golang:1.13-alpine* ?

Also, why 1.13 and not 1.14

Reply:
Buster is because spire is not running inside alpine images. Not found a better solution at the moment.